firebase-sessions/src/main/kotlin/com/google/firebase/sessions/FirebaseSessions.kt

```diff
@@ -32,7 +32,7 @@ internal constructor(
   private val firebaseApp: FirebaseApp,
   internal val backgroundDispatcher: CoroutineDispatcher,
   private val sessionFirelogPublisher: SessionFirelogPublisher,
-  private val sessionGenerator: SessionGenerator,
+  sessionGenerator: SessionGenerator,
```

Comment:
Why is the session generator needed here?

Reply:
We pass it to the SessionInitiator in the init block